ocs_ci/deployment/baremetal.py

```diff
@@ -102,147 +120,182 @@ def deploy_prereq(self):
                 )
-                assert self.helper_node_handler.exec_cmd(cmd=cmd), "Failed to Download required File"
+                assert self.helper_node_handler.exec_cmd(
+                    cmd=cmd
+                ), "Failed to Download required File"
             else:
                 raise RhcosImageNotFound
             # Download installer_kernel
-            kernel_image_path = constants.coreos_url_prefix + image_data['installer_kernel_url']
+            kernel_image_path = (
+                constants.coreos_url_prefix + image_data["installer_kernel_url"]
+            )
             if check_for_rhcos_images(kernel_image_path):
                 cmd = (
                     "wget -O "
                     f"{self.helper_node_details['bm_tftp_dir']}"
                     "/rhcos-installer-kernel-x86_64 "
                     f"{kernel_image_path}"
                 )
-                assert self.helper_node_handler.exec_cmd(cmd=cmd), "Failed to Download required File"
+                assert self.helper_node_handler.exec_cmd(
+                    cmd=cmd
+                ), "Failed to Download required File"
             else:
                 raise RhcosImageNotFound
             # Download metal_bios
-            metal_image_path = constants.coreos_url_prefix + image_data['metal_bios_url']
+            metal_image_path = (
+                constants.coreos_url_prefix + image_data["metal_bios_url"]
+            )
             if check_for_rhcos_images(metal_image_path):
                 cmd = (
                     "wget -O "
                     f"{self.helper_node_details['bm_path_to_upload']}"
                     "/rhcos-metal.x86_64.raw.gz "
                     f"{metal_image_path}"
                 )
-                assert self.helper_node_handler.exec_cmd(cmd=cmd), "Failed to Download required File"
+                assert self.helper_node_handler.exec_cmd(
+                    cmd=cmd
+                ), "Failed to Download required File"
             else:
                 raise RhcosImageNotFound
 
             if ocp_version == "4.6":
                 # Download metal_bios
-                rootfs_image_path = constants.coreos_url_prefix + image_data['live_rootfs_url']
+                rootfs_image_path = (
+                    constants.coreos_url_prefix + image_data["live_rootfs_url"]
+                )
                 if check_for_rhcos_images(rootfs_image_path):
                     cmd = (
                         "wget -O "
                         f"{self.helper_node_details['bm_path_to_upload']}"
                         "/rhcos-live-rootfs.x86_64.img "
                         f"{rootfs_image_path}"
                     )
-                    assert self.helper_node_handler.exec_cmd(cmd=cmd), "Failed to Download required File"
+                    assert self.helper_node_handler.exec_cmd(
+                        cmd=cmd
+                    ), "Failed to Download required File"
```

Comment:
all (4) images have the same code (lines 194 - 256) and can by replace by list of dict :
img_dict = [{"path":  rootfs_image_path = (constants.coreos_url_prefix + image_data["live_rootfs_url"]),
                    "img": "/rhcos-live-rootfs.x86_64.img"}, ... ,{}]
for image in img_dict:
   img_path = image["path"]
   img_url = f"{self.helper_node_details['bm_path_to_upload']}{image['img']"
   cmd = f"wget -o {img_url} {img_path}
   if check_for_rhcos_images(img_path):
      assert self.helper_node_handler.exec_cmd(cmd=cmd), "Failed to Download required File"

Reply:
Just as a general rule I'm trying to avoid changing functionality in this PR. I'm only interested in getting black implemented as an automatic code formatter. This PR will never get merged if I have to fix every issue we have with each of the files black has touched here.